src/templates/static/special/rebootLocalHost.yml

```diff
@@ -0,0 +1,21 @@
+# Reboots a system if any of the preceeding plays sets the 'insights_needs_reboot' variable to true.
+# The variable can be overridden to suppress this behavior.
+- name: Reboot localhost
+  hosts: "@@HOSTS@@"
+  vars:
+    insights_signature_exclude: "/hosts"
+  become: True
+  gather_facts: False
+  tasks:
+    - when:
+        - insights_needs_reboot is defined
+        - insights_needs_reboot
+      block:
+        - name: Reboot system
+          shell: sleep 2 && shutdown -r +1 now "Ansible triggered reboot"
+          async: 1
```

Comment:
IIRC async/poll are needed when the host in question disconnects (do to reboot). In this case we're scheduling it to happen after the playbook finishes and therefore they should not be needed. Same for `sleep 2`